src/components/common/NavBar.js

```diff
@@ -1,101 +1,223 @@
-import React from 'react';
+import React, { useState } from 'react';
 import {
   FacebookOutlined,
   TwitterOutlined,
   InstagramOutlined,
   YoutubeOutlined,
   LinkedinOutlined,
   ThunderboltFilled,
+  HomeOutlined,
+  SnippetsOutlined,
+  AliwangwangOutlined,
+  FormOutlined,
+  BankOutlined,
+  UserSwitchOutlined,
 } from '@ant-design/icons';
+import { Drawer, Menu } from 'antd';
+import {
+  ProfileIcon,
+  HamburgerMenuIcon,
+} from '../pages/LandingInstructor/Icons';
 import { NavLink } from 'react-router-dom';
 
-export default function MainNav() {
+const { SubMenu } = Menu;
+
+const Navbar = () => {
+  const [visible, setVisible] = useState(false);
+
+  const showDrawer = () => {
+    setVisible(true);
+  };
+  const onClose = () => {
+    setVisible(false);
+  };
+
   return (
-    <nav className="mainNav">
-      <div className="nav navLogo">
+    <nav className="navbar__mainNav">
+      <div className="navbar__logo">
         <NavLink to="/">
-          <h1 className="navTitle">
+          <h1 className="navbar__navTitle">
             {
               <ThunderboltFilled
-                style={{ color: '#EEEDD9', fontSize: '2rem' }}
+                style={{
+                  color: '#EEEDD9',
+                  fontSize: '1.8rem',
+                }}
               />
             }{' '}
             CODERHEROES
           </h1>
         </NavLink>
       </div>
-      <div className="nav navOptions">
-        <NavLink className="navLinks" to="/">
+      <div className="navbar__links">
+        <NavLink className="navbar__navLink" to="/">
           PROGRAMS
         </NavLink>
-        <NavLink className="navLinks" to="/browse-instructors">
+        <NavLink className="navbar__navLink" to="/browse-instructors">
           INSTRUCTORS
         </NavLink>
-        <NavLink className="navLinks" to="/parent-booking">
+        <NavLink className="navbar__navLink" to="/parent-booking">
           BOOKING
         </NavLink>
-        <NavLink className="navLinks" to="/">
+        <NavLink className="navbar__navLink" to="/">
           SCHOLARSHIPS
         </NavLink>
       </div>
```

Comment:
Great job with fixing the element names to fit to the naming convention, it will make it much easier to follow when styling.